network-api/networkapi/wagtailpages/models.py

```diff
@@ -556,13 +560,45 @@ class IndexPage(FoundationMetadataPageMixin, Page):
         FieldPanel('intro'),
     ]
 
+    def get_entries(self):
+        return self.get_children().live().order_by('-first_published_at')
+
     def get_context(self, request):
         context = super().get_context(request)
         context = set_main_site_nav_information(self, context, 'Homepage')
         context = get_page_tree_information(self, context)
-        context['entries'] = self.get_children().live().order_by('-first_published_at')
+        context['entries'] = self.get_entries()[0:1]
```

Comment:
realistically we'd probably want this to be `self.get_entries()[0: self.page_size]` for selecting a batch with a CMS-controlled size.

Reply:
It seems like on mobile we show two cards per row and three cards per row on bigger devices. To make new set of cards appear nicely we might wanna choose a number that's a common multiple of 2 and 3 so new cards can appear as full row(s).

Reply:
I'm going to default it to 12, which gets us 2, 3, and 4 (and 6 but pff come on, 6? on a row? not everyone has an ultrawide monitor)